styles/components/_layout.scss

```diff
@@ -257,6 +257,11 @@
   padding: $padLg;
 }
 
+.padBg2,
+.padBgKids > * {
```

Comment:
I'm not seeing padBg here. I think you mean padBg2 should be changed to padLg? Is that correct?